examples/5-nextjs/utils/initModels.ts

```diff
@@ -0,0 +1,24 @@
+
+import { applySnapshot, ModelCreationType } from 'mobx-state-tree';
+import { createHttpClient } from 'mst-gql';
+import { RootStoreType, RootStore } from '../src/models';
+
+let store: ModelCreationType<RootStoreType>;
+
+export function initializeStore (isServer: boolean, snapshot = null): ModelCreationType<RootStoreType> {
+  if (isServer) {
+    store = RootStore.create(undefined, {
+      gqlHttpClient: createHttpClient('http://localhost:4000/graphql'),
+      ssr: true
+    });
+  }
+  if (store === null) {
```

Comment:
Shouldn't this be `store === undefined`? I tried this in a project of mine and it failed because if `store` is not defined, it's not `null`.

Reply:
@fullofcaffeine Yup, you're right. But this example app has been updated and it's no longer an issue. I suggest you refer to the code in `master` (main/default) branch instead: https://github.com/mobxjs/mst-gql/blob/master/examples/5-nextjs/pages/_app.tsx